package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@fontsource/roboto": "^4.5.8",
```

Comment:
The default font is already defined by the global styles branch. Please remove the extra dependency

Reply:
Is this resolved?

Reply:
Dependency removed